skimage/restoration/_rolling_ball.py

```diff
@@ -5,29 +5,30 @@
 
 
 def rolling_ball(image, *, radius=100, kernel=None, nansafe=False, num_threads=None):
-    """Estimate background intensity by rolling/translating a kernel.
+    """Estimate background intensity by translating a kernel over an image.
```

Comment:
```suggestion
    """Estimate background intensity using the rolling ball algorithm.
```

I don't think this is the same as translating a kernel, which sounds more like convolution.

Reply:
Yes, when I started reading the gallery example ("The algorithm works as a filter") and this docstring, I originally thought it was a convolution with a kernel...

Detail: I like hyphenating "rolling-ball algorithm" even if it's not compulsory and most of the literature writes "rolling ball algorithm" (I think it's nicer to read, so my eyes immediately pick up "algorithm" as the main noun and they know "rolling-ball" is just qualifying it).